templates/components/amp/products/card.html

```diff
@@ -8,7 +8,7 @@
             {{/if}}
         {{/or}}
         <a data-vars-product-link="{{url}}" data-vars-product-id="{{id}}" data-vars-product-name="{{name}}" href="{{url}}">
-            <amp-img class="card-image" src="{{getImage image 'productgallery_size' (cdn theme_settings.default_image_product)}}" width="300" height="300" layout="responsive" alt="{{image.alt}}"></amp-img>
+          <amp-img class="card-image" src="{{getImage image 'productgallery_size' (cdn theme_settings.default_image_product)}}" width="{{first (split theme_settings.productgallery_size 'x')}}" height="{{last (split theme_settings.productgallery_size 'x')}}" layout="responsive" alt="{{image.alt}}"></amp-img>
```

Comment:
not familiar with this template language. 
width="{{first (split theme_settings.productgallery_size 'x')}}" height="{{last (split theme_settings.productgallery_size 'x')}}"

not sure what these are doing but is it guaranteed to have a 'first'/'last'? otherwise would you end up with a 0 default height and image wouldn't display?

Reply:
Yeah, we are. This code is pulling values out of the theme configuration, and they are formatted as `${width}x${height}`. For example: https://github.com/bigcommerce/cornerstone/blob/master/config.json#L80